services/core-api/app/api/mines/permits/permit/models/permit.py

```diff
@@ -157,11 +160,16 @@ def find_by_now_application_guid(cls, _now_application_guid):
         return None
 
     @classmethod
-    def create(cls, mine, permit_no, permit_status_code, add_to_session=True):
+    def create(cls, mine, permit_no, permit_status_code, permit_prefix, add_to_session=True):
         permit = cls.find_by_permit_no(permit_no)
+        number_value = None
```

Comment:
Maybe a variable name like `next_permit_no_sequence` is more descriptive of the intention?

Reply:
changed